tests/test_flows.py

```diff
@@ -2609,9 +2611,8 @@ def dog():
         assert flow.name == "dog"
         assert flow.description == "Says woof!"
 
-        # But if the flow is called, it should raise the ScriptError
-        with pytest.raises(ScriptError):
-            flow.fn()
+        # Should still be callable
+        assert flow() == "woof!"
```

Comment:
What happens if the flow uses the non-existent package? Like in this case, what if the flow was:

```
        from not_a_module import not_a_function
        from prefect import flow
        @flow(description="Says woof!")
        def dog():
            return not_a_function('dog')
```

Would the user still get a reasonable error message? This was the original reasoning for the placeholder flow variant, so that a user attempting to execute the flow in an environment where the package was unavailable would be told so.

Reply:
Name error is probably good enough for now, and we can revisit if it's confusing.